src/ucp/wireup/wireup_cm.c

```diff
@@ -357,13 +357,39 @@ ucp_cm_ep_priv_data_pack(ucp_ep_h ep, const ucp_tl_bitmap_t *tl_bitmap,
     return status;
 }
 
+void ucp_wireup_cm_ep_pending_extract(ucp_lane_index_t lane_idx,
+                                      ucp_ep_t *ucp_ep, ucs_queue_head_t *queue)
+{
+    uct_ep_h uct_ep;
+    ucp_wireup_ep_t *wireup_ep;
+    int pending_count;
+
+    /* Old proto selection does not support protocol reset.
+     * With new EP config previous protocol selection might be invalid. */
+    if (!ucp_ep->worker->context->config.ext.proto_enable) {
+        return;
+    }
```

Comment:
BTW according to function name `ucp_wireup_cm_ep_pending_extract` this if-statement should be done out side of the function since it does nothing if protov2 is enabled